scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -752,3 +752,81 @@ object a {
   private[this] val redirectClass =
     pbClass.getClasses find (_.getSimpleName == "Redirect")
 }
+<<< multiline with blanks
+align.preset = most
+===
+object a {
+  def foo: Int = {
+    val x = ???
+    42
+  }
+
+  def bar: Int = {
+    for { x <- "hey" } yield x
+    42
+  }
+
+  def fooBig: Int = {
+    42
+  }
+
+  def fooBigger: Int = {
+    42
+  }
+}
+>>>
+object a {
+  def foo: Int = {
+    val x = ???
+    42
+  }
+
+  def bar: Int = {
+    for { x <- "hey" } yield x
+    42
+  }
+
+  def fooBig: Int = {
+    42
+  }
+
+  def fooBigger: Int = {
+    42
+  }
+}
+<<< multiline without blanks
+align.preset = most
+===
+object a {
+  def foo: Int = {
+    val x = ???
+    42
+  }
+  def bar: Int = {
+    for { x <- "hey" } yield x
+    42
+  }
+  def fooBig: Int = {
+    42
+  }
+  def fooBigger: Int = {
+    42
+  }
+}
+>>>
+object a {
+  def foo: Int       = {
```

Comment:
Is it possible to flush blocks for this case too? I don't think that anyone wants to align `=` owned by function definition

Reply:
> Is it possible to flush blocks for this case too? I don't think that anyone wants to align `=` owned by function definition

there's a case in the tests supporting this:
```
  private[this] val pbClass       = Class.forName("java.lang.ProcessBuilder")
  private[this] val redirectClass =
    pbClass.getClasses find (_.getSimpleName == "Redirect")
```

do you want to restrict just `Defn.Def` and not `Defn.Val`? what about `def` instead of `val` in that example:
```
  private[this] def pbClass       = Class.forName("java.lang.ProcessBuilder")
  private[this] def redirectClass =
    pbClass.getClasses find (_.getSimpleName == "Redirect")
```

Reply:
@poslegm if you don't mind, i will merge this as-is, and then send you a patch for review to try to figure out what to do about `=` in definitions.

Reply:
> do you want to restrict just Defn.Def and not Defn.Val?

I just forgot that scalafmt aligns one-line defs  by `=` already. So, it's OK and consistent to align also multilne 👍